contract-deployment/contractDeploymentTools.go

```diff
@@ -4,143 +4,15 @@
 package main
 
 import (
-	"encoding/hex"
-	"encoding/json"
 	"fmt"
-	"io/fs"
 	"log"
-	"math/big"
 	"os"
 	"strconv"
 
-	"github.com/ava-labs/subnet-evm/core/types"
+	deployment_utils "github.com/ava-labs/teleporter/contract-deployment/deployment-utils"
```

Comment:
I think we should just name the pkg `utils`, and places that import can rename if needed.